lldb/source/Plugins/Language/ObjCPlusPlus/ObjCPlusPlusLanguage.h

```diff
@@ -27,6 +27,8 @@ class ObjCPlusPlusLanguage : public Language {
     return lldb::eLanguageTypeObjC_plus_plus;
   }
 
+  const char *GetUserEntryPointName() const override { return "main"; }
```

Comment:
Unrelated to this PR but it would have been nice if this class was derived from the `CPlusPlus` and/or `ObjC` one so you wouldn't have to re-implement the same method here. @Michael137 do you think that it could be a useful refactor ? Or may be we should introduce a `C` language base class that every thing derives from ?